scripts/server.sh

```diff
@@ -4,14 +4,6 @@ set -e
 
 echo "Run the RWD Docker container."
 
-if [ -z "$RWD_DATA" ]; then
```

Comment:
I think receiving an error message when `RWD_DATA` is empty or undefined is the correct behavior. Let's keep these as-is for now and tweak it if we run into any problems.